designer/server/src/main/scala/pl/touk/nussknacker/ui/validation/UIProcessValidator.scala

```diff
@@ -24,20 +23,20 @@ import pl.touk.nussknacker.ui.process.fragment.FragmentResolver
 import pl.touk.nussknacker.ui.process.marshall.ProcessConverter
 import pl.touk.nussknacker.ui.process.processingtypedata.ProcessingTypeDataProvider
 
-object ProcessValidation {
+object UIProcessValidator {
```

Comment:
This also contains some functions operating on CanonicalProcess like `processingTypeValidationWithTypingInfo` . This could be confusing since in my mind UIProcess = DisplayableProcess. Maybe extract that function to another obj like `CanonicalProcessValidator` or leave a TODO?